src/Containers/Microsoft.NET.Build.Containers/ManifestListV2.cs

```diff
@@ -10,3 +10,5 @@ public record struct ManifestListV2(int schemaVersion, string mediaType, Platfor
 public record struct PlatformInformation(string architecture, string os, string? variant, string[] features, [property: JsonPropertyName("os.version")][field: JsonPropertyName("os.version")] string? version);
 
 public record struct PlatformSpecificManifest(string mediaType, long size, string digest, PlatformInformation platform);
+
+public record struct ImageIndexV1(int schemaVersion, string mediaType, PlatformSpecificManifest[] manifests);
```

Comment:
ManifestListV2 has the same structure. I think we can reuse it instead of adding a new struct

Reply:
I thought about that but I wanted to make the new structure to for two reasons:

* even though they structurally share fields, they are different concepts, so I wanted to be able to use the different structure as an input to different methods to ensure methods know which kind of manifest they were dealing with
* there are more fields than just these three on the OCI Image Index, this is just the required fields to read the index and do RID inference. For example, annotations are supported on Image Indexes, so if we have this structure ready to go then adding `Dictionary<string, string> annotations` becomes a very small change in the future.